src/components/SummaryKeyComponents/SelectionSummaryComponent.js

```diff
@@ -0,0 +1,50 @@
+import { textSummarizer } from "../../services/SummarizerService";
+import {
+  EventHandlerService,
+  PDFLEvents,
+} from "../../services/EventHandlerService";
+import { SelectionPopUpComponent } from "../SelectionPopUpComponent";
+
+/**
+ * Class representing the Selection Summary Accordion item
+ * @property {Object} components object that holds DOM elements that represent this component, as well as component's context
+ * @property {HTMLElement} components.selectedSummaryText html p element that represents the text of the accordion item
+
+ */
+class SelectionSummaryComponent {
```

Comment:
Yeah I was thinking the same thing because all of these elements
![image](https://user-images.githubusercontent.com/79309587/209596491-8d8ec370-b2ee-4685-8973-cc5676f23499.png)
are pretty much the same component/element with different text displayed and event to set it.